app/src/main/java/org/apache/fineract/ui/online/login/LoginActivity.java

```diff
@@ -50,6 +51,7 @@ protected void onCreate(Bundle savedInstanceState) {
         loginPresenter.attachView(this);
         setActionBarTitle(getString(R.string.fineract_account));
         preferencesHelper.clear();
+        getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_RESIZE);
```

Comment:
Instead, of making changes here, make changes in AndroidManifest.xml

Reply:
@miPlodder Done.